paper-slider.html

```diff
@@ -472,8 +476,8 @@
       },
 
       keyBindings: {
-        'left down pagedown home': '_decrementKey',
-        'right up pageup end': '_incrementKey'
+        'left down pagedown home': '_leftKey',
+        'right up pageup end': '_rightKey'
```

Comment:
page/down/up keys should not be reversed in RTL (e.g. down still means decrement). Makes sense to split each key binding to a separate handler:

```javascript
'left': '_leftKey',
'right': '_rightKey',
'down pagedown home': '_decrementKey',
'up pageup end': '_incrementKey'
```